### ISSUE TITLE
Unable to assign config values that expect a List

### ISSUE THREAD

**Issue by bravetarget** [wavesplatform/node-docker-image#2]
**Describe the bug**
Defining a node configuration list/array value from the environment variables fails on launch

**To Reproduce**
1. Define `WAVES__FEATURES__SUPPORTED` to be `[1,2]` in the environment variables
2. Run container...
3. See error

**Expected behavior**
`waves.features.supported` should have an array value of `[1,2]` in `local.conf` 

**Error Log**
```
Exception in thread "main" com.typesafe.config.ConfigException$WrongType: /waves/configs/local.conf: 12: waves.features.supported has type STRING rather than LIST
```

**Additional context**
This is non-critical, since most of the configuration items that actually need redefining are handled with strings.

**Comment by KardanovIR** [comment]
@bravetarget thank you for your report! This issue should be fixed with 7a647b7a23add2fb7564942d5e5385efa269152f. Can you check it again, please?

**Comment by bravetarget** [comment]
@KardanovIR cool!  tested arrays in env injected config with https://github.com/wavesplatform/node-docker-image/commit/7a647b7a23add2fb7564942d5e5385efa269152f and that fixes it.  :tada:
